src/constants.ts

```diff
@@ -103,12 +103,18 @@ export class Constants {
     public static IoTHubAIDGetModuleDoneEvent = "AZ.Module.Get.Done";
     public static IoTHubAIUpdateDistributedSettingStartEvent = "AZ.DistributedTracing.Update.Start";
     public static IoTHubAIUpdateDistributedSettingDoneEvent = "AZ.DistributedTracing.Update.Done";
-    public static IoTHubAILoadDistributedTracingSettingStartEvent = "AZ.LoadDistributedTracingSetting.Start";
-    public static IoTHubAILoadDistributedTracingSettingDoneEvent = "AZ.LoadDistributedTracingSetting.Done";    public static IoTHubAIStartLoadDeviceTreeEvent = "General.StartLoadDeviceTree";
+    public static IoTHubAILoadDistributedTracingSettingTreeStartEvent = "AZ.LoadDistributedTracingSettingTree.Start";
+    public static IoTHubAILoadDistributedTracingSettingTreeDoneEvent = "AZ.LoadDistributedTracingSettingTree.Done";
+    public static IoTHubAIStartLoadDeviceTreeEvent = "General.StartLoadDeviceTree";
     public static IoTHubAIShowWelcomePagetEvent = "General.WelcomePage.Show";
+    public static PercentageLabel = "(%)";

```

Comment:
In English world, it seems 100% is more than 100(%)

Reply:
Because we want to make user not have to guess if he should input [int] or [int]%